swift/StableDiffusion/pipeline/Decoder.swift

```diff
@@ -36,13 +36,16 @@ public struct Decoder: ResourceManaging {
     ///  - Parameters:
     ///    - latents: Batch of latent samples to decode
```

Comment:
Thanks for the review @alejandro-isaza! Since you approved with a single comment, I will merge this now and push another commit to address the new parameter right after 👍